### PR TITLE
fix website breaking on note delete

### LINKED ISSUE #56: After Deleting a note and returning to Home page the website breaks
Steps to produce error:

Delete a already created note
Shift back or directly go to home page
BUG
The Redux is unable to fetch the data for all notes

### PR DESCRIPTION
**Pull request by ozair-dev**
I made some changes and I think I fixed the issue.
Please deploy my changes so I can see if the any issue remains.

Fixes #56

### CODE REVIEW

**Comment by ozair-dev** [comment]
u there?

**Comment by Prasundas99** [comment]
Hi, didn't realized a PR was created let me check

File: client/src/redux/reducers/NotesReducer.js [hunk 1]
```diff
@@ -1,38 +1,43 @@
...
-  data: {}
-}
+  data: {},
+};
 
 const Notes = (state = initialstate, action) => {
   switch (action.type) {
     case FETCH_ALL:
-      return {notes: action.payload};
+      return { ...state, notes: action.payload };
```
Comment (Prasundas99) [inline client/src/redux/reducers/NotesReducer.js:17]: Ah! so this was the main culprit great work

File: client/src/redux/actions/userAction.js [hunk 2]
```diff
@@ -20,10 +20,10 @@ export const userRegister = (username, email, password) => async (dispatch) => {
     const config = {
       "Content-Type": "application/json",
     };
-    const baseUrl="https://mydiary-backend-host.herokuapp.com";
+    const baseUrl = "https://mydiary-backend-host.herokuapp.com";
```
Comment (Prasundas99) [inline client/src/redux/actions/userAction.js:23]: I'll prefer this to be used as a global variable.

Create a const.js add these baseurl there

File: client/src/redux/actions/userAction.js [hunk 3]
```diff
@@ -20,10 +20,10 @@ export const userRegister = (username, email, password) => async (dispatch) => {
...
     const { data } = await axios.post(
-          baseUrl+"/users/signup",
-            { email, password, username },
+      baseUrl + "/users/signup",
```
Comment (Prasundas99) [inline client/src/redux/actions/userAction.js:25]: create a const vaiable which will be used in that const.js


```
const baseUrl = "https://mydiary-backend-host.herokuapp.com"

const apiRoutes = {
    register = '/users/signup'
}

```

and here :

`axios.post(${baseUrl + apiRoute.register})`

File: client/src/redux/actions/userAction.js [hunk 4]
```diff
@@ -52,10 +52,10 @@ export const userLogin = (email, password) => async (dispatch, getState) => {
     const config = {
       "Content-Type": "application/json",
     };
-
+    const baseUrl = "https://mydiary-backend-host.herokuapp.com";
```
Comment (Prasundas99) [inline client/src/redux/actions/userAction.js:55]: This one too

**Review by Prasundas99 — CHANGES_REQUESTED** [review]

**Comment by Prasundas99** [comment]
> u there?

Hi, @ozair-dev thank you for the PR I have added a few more changes which will help to maintain the code. Can you please add those otherwise your PR seems good to me

**Pull request closed without merging**